certif/tests/acceptance/session-import_test.js

```diff
@@ -211,6 +211,44 @@ module('Acceptance | Session Import', function (hooks) {
             // then
             assert.dom(screen.getByRole('button', { name: 'Finaliser la création/édition' })).exists();
           });
+
+          module('when the user has confirmed the import', function () {
+            test("it should redirect to the session's list", async function (assert) {
```

Comment:
Suggestion: splitter en 2 tests ? 

- it should redirect to the session's list

- it should display a success notification